### PR TITLE
IOS-47 fix

### LINKED ISSUE #47: Unresponsive Vote Screen on iOS 11
Above-mentioned screen becomes unresponsive after dismissing AlertView. This happens regardless of my last changes.

### PR DESCRIPTION
**Pull request by sxcore**
Fixes #47

### CODE REVIEW

**Comment by Drwalcore** [comment]
[iOS - 47](https://trello.com/c/kewMxvOR/111-ios-47)

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
-# Xcode
-#
-# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
+## Xcode
+Swift.gitignore
```
Comment (mateuszszklarek) [inline .gitignore]: Please remove this line, and paste below gitignore:

```
## Build generated
build/
DerivedData/

## Various settings
*.pbxuser
!default.pbxuser
*.mode1v3
!default.mode1v3
*.mode2v3
!default.mode2v3
*.perspectivev3
!default.perspectivev3
xcuserdata/
!*.xcworkspace/contents.xcworkspacedata

## Other
*.moved-aside
*.xcuserstate

## Sourcery
default.profraw

## Obj-C/Swift specific
*.hmap
*.ipa
*.dSYM.zip
*.dSYM

## Playgrounds
timeline.xctimeline
playground.xcworkspace

## Packages/
.build/

## CocoaPods
Pods/

## Carthage
Carthage/Build

## fastlane
fastlane/report.xml
fastlane/Preview.html
fastlane/screenshots
fastlane/test_output

## FBSnapshotTestCase
FailureDiffs/

## Deployment scripts
.setup_travis_deployment.sh
.match_key

## AppCode
.idea/
```

**Review by mateuszszklarek — CHANGES_REQUESTED** [review]
Please update gitignore.

**Review by mateuszszklarek — APPROVED** [review]
Mergeable 👍

**Pull request merged**